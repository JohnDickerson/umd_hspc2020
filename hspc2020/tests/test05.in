10

STANLEY
NLY

ADOBECODEBANC
BANC

KELLYKAPOOR
YP

KELLYKAPOOR
KR

KELLYKAPOOR
KELLY

STANLEYHUDSON
LD

STANLEYHUDSON
HN

DWIGHTSCHRUTE
DW

DWIGHTSCHRUTE
DIGT

DWIGHTSCHRUTE
DIGE




