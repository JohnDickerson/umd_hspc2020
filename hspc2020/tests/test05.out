NLEY
BANC
YKAP
KAPOOR
KELLY
LEYHUD
NLEYH
DW
DWIGHT
DWIGHTSCHRUTE
